oap-server/server-starter/src/main/resources/component-libraries.yml

```
@@ -374,6 +374,9 @@ Sentinel:
 GuavaCache:
   id: 114
   languages: Java
+druid:
```

```suggestion
AlibabaDruid:
```

According to https://github.com/apache/skywalking-java/pull/27/files#r711735928